test/lit/passes/generate-dyncalls-wasm64.wast

```diff
@@ -0,0 +1,52 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; RUN: wasm-opt %s --generate-dyncalls --enable-memory64 --table64-lowering -S -o - | filecheck %s
```

Comment:
Why add lowering here?

Reply:
Hmm.. I think you are right.. I need to make sure that it still fails without that patch in that case.

Reply:
Yup, you are correct.  The orignal crash and the fix both confirmed without the need for lowering here.